src/Athena/Views/Home/Main_Page.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Main_Page";
+}
+<br /><br /><br /><br /><br />
+
+<img src="~/images/UT-color-vert-trans.gif" class="responsive-img left" style="height:10em; max-width:100%" />
+<br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
+
+
+    <h4>Degree Progress:</h4>
+    <br />
+    <div id="progress-bar" class="all-rounded" style="height:2em;width:10em">
+        <div id="progress-bar-percentage" class="all-rounded" style="width:68%; height:2em"><span>68/100</span></div>
+    </div>
+    <div>
+        <h5>ETC:</h5>??/??/????
+    </div>
+
+<br />
+
+<h4>Class Schedule:</h4>
+<div id='calendar'></div>
+
+<a href="Schedule_Page" style="color:white" class="btn">New Schedule</a>
+
+<div class="btn">
+    print
+</div>
+
+<a href="Add_Classes" style="color:white" class="btn">Complete Classes</a>
```

Comment:
Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.